pages/search.vue

```diff
@@ -195,17 +195,12 @@
             placeholder="Choose versions..."
             @input="onSearchChange(1)"
           ></multiselect>
-          <h3 class="sidebar-menu-heading">Licenses</h3>
-          <Multiselect
-            v-model="selectedLicenses"
-            placeholder="Choose licenses..."
-            :loading="$tag.licenses.length === 0"
-            :options="$tag.licenses.map((x) => x.short.toUpperCase())"
-            :multiple="true"
-            :searchable="true"
-            :close-on-select="false"
-            :show-labels="false"
-            :allow-empty="true"
+          <h3 class="sidebar-menu-heading">Open source</h3>
+          <Checkbox
+            v-model="onlyOpenSource"
+            label="Only include open source projects"
```

Comment:
The label is a little too verbose IMO. "Open source" should suffice. The other category filters aren't phrased this weird so it seems a little out of place.

Reply:
The main issue is that, with that phrasing, it will imply that open source projects are excluded if it's unselected by default, whereas this phrasing makes it explicit that it's excluding non-open-source projects. Would "Open source only" be good?

Reply:
Does it? For the categories it doesn't seem that way. "Open source only" is good too but I think either works. Like we don't have "Adventure only" or "Fabric only".